identity-admin/webapp/components/search/StatusDropdown.tsx

```diff
@@ -8,19 +8,64 @@ const StatusDropdown = (): JSX.Element => {
   const [statusValue] = useState<string>(
     typeof status === 'string' ? status || '' : ''
   );
+  const [isExpanded, setIsExpanded] = useState<boolean>(false);
 
-  const onChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
-    window.location.href = buildSearchUrl('1', event.target.value, name, email);
+  const toggleExpansion = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  const statuses: { key: string; value: string }[] = [
+    {
+      key: 'any',
+      value: 'Any',
+    },
+    {
+      key: 'active',
+      value: 'Active',
+    },
+    {
+      key: 'locked',
+      value: 'Blocked',
+    },
+    {
+      key: 'deletePending',
+      value: 'Pending delete',
+    },
+  ];
+
+  const statusLabel = (): string => {
+    return (
+      statuses
+        .filter(s => s.key === statusValue)
+        .map(s => s.value)
+        .pop() || 'Select status'
+    );
   };
 
   return (
-    <select onChange={onChange} value={statusValue}>
-      <option hidden>Select status</option>
-      <option value="any">Any</option>
-      <option value="active">Active</option>
-      <option value="locked">Blocked</option>
-      <option value="deletePending">Pending Delete</option>
-    </select>
+    <div className="status-dropdown">
+      <div className="status-dropdown__label" onClick={toggleExpansion}>
+        <span>{statusLabel()}</span>
+        <span className="status-dropdown__label-arrow">
+          {isExpanded ? '▴' : '▾'}
+        </span>
+      </div>
+      {isExpanded && (
+        <div className="status-dropdown__options">
+          {statuses.map((s, i) => {
```

Comment:
The `i` is fine, but can the single-letter variable `s` be made clearer?

```suggestion
          {statuses.map((status, i) => {
```